Fix flaky TestDB_UpdatedAt test

Reported by Marin on the Litestream Slack:

```
=== RUN   TestDB_UpdatedAt/WAL
    db_test.go:111: expected newer updated at time: 2021-03-07 22:12:44.626788764 +0000 UTC > 2021-03-07 22:12:44.626788764 +0000 UTC
--- FAIL: TestDB_UpdatedAt (0.01s)
    --- PASS: TestDB_UpdatedAt/ErrNotExist (0.00s)
    --- PASS: TestDB_UpdatedAt/DB (0.00s)
    --- FAIL: TestDB_UpdatedAt/WAL (0.00s)
```

--- a/db_test.go
+++ b/db_test.go
@@ -100,9 +100,12 @@ func TestDB_UpdatedAt(t *testing.T) {
 			t.Fatal(err)
 		}
 
+		sleepTime := 100 * time.Millisecond
 		if os.Getenv("CI") != "" {
-			time.Sleep(1 * time.Second)
+			sleepTime = 1 * time.Second
 		}
+		time.Sleep(sleepTime)
+
 		if _, err := sqldb.Exec(`CREATE TABLE t (id INT);`); err != nil {
 			t.Fatal(err)
 		}

